scalardb-sample/database.properties

```diff
@@ -1,4 +1,29 @@
-scalar.db.storage=cassandra
-scalar.db.contact_points=localhost
-scalar.db.username=cassandra
-scalar.db.password=cassandra
+# For MySQL
+# scalar.db.storage=jdbc
+# scalar.db.contact_points=jdbc:mysql://localhost:3306/
+# scalar.db.username=root
+# scalar.db.password=mysql
+
+# For PostgreSQL.
+# scalar.db.storage=jdbc
+# scalar.db.contact_points=jdbc:postgresql://localhost:5432/
```

Comment:
```suggestion
# scalar.db.contact_points=jdbc:postgresql://localhost:5432/postgres
```
From the perspective of PostgreSQL specification, I think we need to the database name to access in the JDBC URL. And, I think it's fine to use the default database name `postgres` here.

Reply:
Ah, sorry. I misunderstood something. It seems that we can run ScalarDB without the database name in the JDBC URL. I will close (resolve) this conversation. 🙇‍♀️